Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -195,21 +199,34 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
 
         switch (this._type) {
         case WI.InlineSwatch.Type.Color:
+            let title = "";
+            if (this._swatchTooltip)
+                title += this._swatchTooltip;
             if (!this._readOnly) {
-                let title = WI.UIString("Click to select a color.");
+                title = WI.UIString("Click to select a color.");
                 if (this._allowChangingColorFormats())
                     title += "\n" + WI.UIString("Shift-click to switch color formats.");
                 if (InspectorFrontendHost.canPickColorFromScreen())
                     title += "\n" + WI.UIString("Option-click to pick color from screen.");
                 
                 this._swatchElement.title = title;
             }
+            if (title.length)
+                this._swatchElement.title = title;
             // fallthrough
 
         case WI.InlineSwatch.Type.Gradient:
             this._swatchInnerElement.style.background = value ? value.toString() : null;
             break;
 
+        case WI.InlineSwatch.Type.Variable:
+            if (!this._subType)
+                break;
+            let pallet = (this._colorPalettes.find(pallet => value.toString().includes(pallet.colorSource)));
```

Comment:
we have to use includes because the value is something like "var(--XXX)" and the actual pallet.colorSource is just --XXX so I used include. I had thought of something like `var(${colorPallet.colorSoruce})` and then using ===. What do you think ?